src/Tests/Microsoft.NET.Publish.Tests/RuntimeIdentifiersTests.cs

```diff
@@ -79,6 +83,55 @@ public void BuildWithRuntimeIdentifier()
             }
         }
 
+        //  Run on core MSBuild only as using a local packages folder hits long path issues on full MSBuild
+        [CoreMSBuildOnlyFact]
```

Comment:
I don't think this test needs to set the RestorePackagesPath.  Some of the other tests in this class do for different reasons.  So I'd suggest removing the RestorePackagesPath and switching to a normal `Fact` attribute.